service/test/_job_client_test.js

```diff
@@ -234,10 +234,10 @@ describe('JobClient', function() {
     });
   });
 
-  describe('cancelJob', function() {
+  describe('#cancelJob', function() {
     /*Tests_SRS_NODE_JOB_CLIENT_16_008: [The `cancelJob` method shall throw a `ReferenceError` if `jobId` is `null`, `undefined` or an empty string.]*/
     [undefined, null, ''].forEach(function(badValue) {
-      testFalsyArg(new JobClient({}).cancelJob, 'jobId', badValue, [badValue]);
+      testFalsyArg(new JobClient({}).cancelJob, 'jobId', badValue, [badValue, () => {}]);
```

Comment:
>, () => {}] [](start = 76, length = 11)

And here.  Why added?

Reply:
Done.

---
In reply to: [222828586](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r222828586) [](ancestors = 222828586)